src/@batch-flask/ui/buttons/clickable/clickable.component.ts

```diff
@@ -95,6 +104,7 @@ export class ClickableComponent implements OnChanges, OnDestroy {
 
     @HostListener("click", ["$event"])
     public handleClick(event: Event) {
+        this.selected = true;
```

Comment:
What happens when multiple elements on the same page get selected?

Reply:
They all display the 'selected' attribute

Reply:
If each of the options were selected at one point and now all of them display 'selected', what if the user wanted to go through and see which options were available before going back and choosing one? How would they know which one is actively being selected on the page?

Reply:
I now see that I misinterpreted what exactly the ‘selected’ feature was supposed to do. I just fixed it so that **only** the selected element has the value _true_ for aria-selected and all other elements have a value of _false_ for aria-selected. I will update this PR shortly.